tests/modules/programs/aerc/aerc-2.nix

```diff
@@ -0,0 +1,39 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  imports = [ ../../accounts/email-test-accounts.nix ];
+
+  config = {
+    accounts.email.accounts = {
+      "hm@example.com" = {
+        primary = true;
+        msmtp.enable = true;
+        neomutt = {
+          enable = true;
+          extraConfig = ''
+            color status cyan default
+          '';
+        };
+        imap.port = 993;
+      };
+    };
+
+    programs.neomutt.enable = true;
+
+    nixpkgs.overlays =
+      [ (self: super: { neomutt = pkgs.writeScriptBin "dummy-neomutt" ""; }) ];
```

Comment:
We also need an overlay for notmuch, aerc and msmtp.

Also, the test needs to check for aerc configuration files.